config.py

```diff
@@ -15,12 +15,14 @@ class Config(object):
     MAIL_PASSWORD = getenv('MAIL_PASSWORD')
     ADMINS = [getenv('MAIL_USERNAME')]
 
+
 class DevConfig(Config):
     SECRET_KEY = '4f\g45t45gfjerkfefker'
     SECURITY_PASSWORD_SALT = 'my_precious_two'
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     SQLALCHEMY_POOL_TIMEOUT = 30
     DEBUG = True
+    TESTING = True
```

Comment:
Why you added that?

Reply:
With that condition app don't send mail but only mimics it. That's why it's possible do give false credentials in .env without causing error.

Reply:
We should have separate config for testing and don't mix that with dev config.